tests/src/Kernel/Entity/OgMembershipReferenceItemListTest.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\system\Tests\Entity\OgMembershipReferenceItemTest.
+ */
+
+namespace Drupal\Tests\og\Kernel\Entity;
+
+use Drupal\Component\Utility\Unicode;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\og\Entity\OgMembership;
+use Drupal\og\Og;
+use Drupal\og\OgMembershipInterface;
+
+/**
+ * Tests OgMembershipReferenceItem and OgMembershipReferenceItemList classes.
+ *
+ * @group Entity
+ * @coversDefaultClass \Drupal\og\Plugin\Field\FieldType\OgMembershipReferenceItemList
+ */
+class OgMembershipReferenceItemListTest extends KernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = ['entity_test', 'user', 'field', 'entity_reference', 'og', 'system'];
+
+  /**
+   * Array with the bundle IDs.
+   *
+   * @var array
+   */
+  protected $bundles;
+  protected $fieldName;
+  protected $groups;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    // Add membership and config schema.
+    $this->installConfig(['og']);
+    $this->installEntitySchema('entity_test');
+    $this->installEntitySchema('og_membership');
+    $this->installEntitySchema('user');
+
+    // Create several bundles.
+    for ($i = 0; $i <= 4; $i++) {
+      $bundle = EntityTest::create([
+        'type' => Unicode::strtolower($this->randomMachineName()),
+        'name' => $this->randomString(),
+      ]);
+
+      $bundle->save();
+      $this->bundles[] = $bundle->id();
+    }
+    for ($i = 0 ; $i < 2; $i++) {
+      $bundle = $this->bundles[$i];
+      Og::groupManager()->addGroup('entity_test', $bundle);
+      $group = EntityTest::create(['type' => $bundle]);
+      $group->save();
+      $this->groups[] = $group;
+    }
+    $this->fieldName = strtolower($this->randomMachineName());
+
+    FieldStorageConfig::create([
```

Comment:
`field_name` is not copied down as I mentioned. And it's useless because you need to pass the *exact same array* to a one off helper as to the well known documented and used core ::create functions.